bom/pom.xml

```diff
@@ -319,11 +319,6 @@ THE SOFTWARE.
         <artifactId>symbol-annotation</artifactId>
         <version>1.1</version>
       </dependency>
-      <dependency>
-        <groupId>com.sun.mail</groupId>
-        <artifactId>jakarta.mail</artifactId>
-        <version>1.6.5</version>
```

Comment:
Since this confused me: `WEB-INF/lib/jakarta.mail-1.6.5.jar` actually contains `javax.mail` (in addition to `com.sun.mail`) despite the artifact name. https://eclipse-ee4j.github.io/mail/docs/JavaMail-2.0-changes.txt explains

> Previous version of the Jakarta Mail Specification which is based on
> JavaMail 1.6 used javax.mail package namespace.
> New Jakarta Mail Specification uses jakarta.mail package namespace instead.

Reply:
Correct